sensor/common/collectorruntimeconfig/handler.go

```diff
@@ -119,6 +123,16 @@ func (h *handlerImpl) pushConfigToValueStream() {
 	h.collectorConfigProtoStream.Push(h.collectorConfig)
 }
 
+func (h *handlerImpl) SetCollectorConfigFromHelmConfig(helmManagedConfig *central.HelmManagedConfigInit) {
+	if helmManagedConfig != nil {
+		h.collectorConfig = &sensor.CollectorConfig{
```

Comment:
Why is this struct a property on the handler? I think this must only be passed to Collector, so pushing it to the value stream is sufficient?